src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
add class star for all parent div